examples/python-api-flask/.saturn/saturn.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "example-api-flask",
+  "image_uri": "public.ecr.aws/saturncloud/saturn:2022.01.06",
+  "description": "Deploy a flask API on Saturn Cloud.",
```

None of our other descriptions have periods at the end :)

```suggestion
  "description": "Deploy a flask API on Saturn Cloud",
```